mpa/cls/trainer.py

```diff
@@ -65,7 +65,9 @@ def run(self, model_cfg, model_ckpt, data_cfg, **kwargs):
                     dash_line)
 
         # Data
-        if 'unlabeled' in cfg.data:
+        # FIXME[Soobee] : if train type is not specified, it raises an error in default INCREMENTAL mode.
+        # Should move and refactor in stage.py importing otx.api module
+        if cfg.train_type == 'SEMISUPERVISED':
```

Comment:
No, we don't always have unlabeled. cfg.data cannot have unlabeled when its dataset item entity is empty. but as you said, even user specified 'unlabeled-root-data', we should run INCREMENTAL otherwise train_type is exactly specified as SEMISUPERVISED. In that case, 'unlabeled' would be inside cfg.data even if train_type is INCREMENTAL.